frontend/shared-ui/assets/styles/main.scss

```
@@ -4,6 +4,12 @@
 
 :root {
   font-size: 16px;
+  // unovis injects its own :root font stack at runtime, after this stylesheet.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)